styles.css

```diff
@@ -127,3 +132,123 @@ body {
   opacity: 1;
   width: 100%;
 }
+
+@media (min-width: 1900) {
+
+  .wrapper {
+    border: 10px solid red;
+  }
+
+}
+
+@media (max-width: 768px) {
+  #checkbox {
+    opacity: 1;
+  }
+
+  #checkbox:checked~.burger::after {
+    transform: rotate(135deg);
+    margin-top: -3px;
+  }
+
+  #checkbox:checked~.burger::before {
+    transform: rotate(45deg);
+  }
+
+  #checkbox:checked~.burger {
+    border: none;
+  }
+
+  #checkbox:checked~.menu {
+    display: block;
+    text-align: center;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6278784)